### PR TITLE
Add e2e test for User's last login

### PR DESCRIPTION
**Pull request by Piotrk39**
[AB#30874](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/30874) 

## Describe your changes

Adds test that asserts login time stamps in Users page

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

**Comment by Piotrk39** [comment]
For the reviewer: PR also includes another test "View users page"

File: e2e/portalicious/pages/UsersPage.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    );
+
+    expect(sortedActualUserEmails).toEqual(sortedExpectedUserEmails);
+  }
+
+  async validateRowTextContent({
+    email,
+    textContent,
+  }: {
+    email?: string;
```
Comment (aberonni) [inline e2e/portalicious/pages/UsersPage.ts:54]:
```suggestion
    email: string;
```

I don't think this should be optional, right?

**Review by aberonni — CHANGES_REQUESTED** [review]
Small change but otherwise LGTM!

File: e2e/portalicious/pages/UsersPage.ts (hunk 1 above)
Comment (Piotrk39) [reply e2e/portalicious/pages/UsersPage.ts]: Yes it should not. removed that and I fixed the type issue in the test. Thanks!

**Review by aberonni — APPROVED** [review]

**Pull request merged**